tests/tracer/test_propagation.py

```diff
@@ -1111,7 +1143,7 @@ def test_propagation_extract(name, styles, headers, expected_context, run_python
     ),
     (
         "valid_all_styles_no_sampling_priority",
-        PROPAGATION_STYLE_ALL,
```

Comment:
Question: are we removing PROPAGATION_STYLE_ALL from all tests? Should we use it in some tests?

Reply:
We're just replacing it with the what `PROPAGATION_STYLE_ALL` is explicitly so that it can be examined how order matters more easily. Since a user can't set `PROPAGATION_STYLE_ALL`  but would rather need to set `datadog, b3, b3 single header` explicitly, I think it's fine to not test with it. Does that make sense/seem right?